README.rst

```diff
@@ -85,18 +89,86 @@ Next, you must create a user with API access:
        LIMESURVEY_API_USER = "<username>"
        LIMESURVEY_API_PASSWORD = "<password>"
 
+
+Configuring LimeSurvey
+**********************
+
+Survey Modes
+============
+In LimeSurvey, you can configure 2 survey modes: closed or open (anonymous).
+
+- **Closed:** Closed surveys limit access to the survey to any person, i.e., only students with
+  an access code will be able to fill it out.
+- **Open:** Open surveys allow any student with access to the link to fill out the survey. In this mode,
+  there is no way to relate the answers to the students.
+
+
+Recommendations
+===============
+- All surveys added of closed-access mode, must have an ``attribute_1``, which
+  allows the assignment of a unique identifier for each survey participant. If this attribute
+  is not added, students will not be able to complete the survey.
```

Comment:
Instead of adding a recommendations section, we can add in the closed description, don't you think?